app/models/requests/CnCodeInformationRequest.scala

```diff
@@ -27,5 +28,8 @@ object CnCodeInformationRequest {
 }
 
 object CnCodeInformationItem {
+  def apply(items: Seq[ItemsAddToListItemModel]): Seq[CnCodeInformationItem] =
+    items.map(item => CnCodeInformationItem(item.exciseProductCode, item.commodityCode))
```

Comment:
Looks like this isn't shown on the page

Reply:
I think so, I'll add it in to be consistent. It's something I brought up on Item CYA and UCD weren't sure if we'd have access to this info so it wasn't included in the copydeck etc, but since we do have access to it, I assume they'll want it added.

Reply:
In fact that can be an alignment, I can't currently think of a clean way to do this.